sphinx/writers/latex.py

```diff
@@ -517,12 +517,13 @@ def generate(content: list[tuple[str, list[IndexEntry]]], collapsed: bool) -> No
     def render(self, template_name: str, variables: dict[str, Any]) -> str:
         renderer = LaTeXRenderer(latex_engine=self.config.latex_engine)
         for template_dir in self.config.templates_path:
-            template = path.join(self.builder.confdir, template_dir,
-                                 template_name)
-            if path.exists(template):
-                return renderer.render(template, variables)
+            for template_suffix in ('_t', '.jinja'):
+                template = path.join(self.builder.confdir, template_dir,
+                                     template_name + template_suffix)
+                if path.exists(template):
+                    return renderer.render(template, variables)
 
-        return renderer.render(template_name, variables)
+        return renderer.render(template_name + '_t', variables)
```

Comment:
Something about this line feels wrong to me, in the sense of introducing an unintended behaviour change / incompatibility.

Reply:
I think it's safest to undo this and further reduce the size of the diff.